### PR TITLE
[VBLOCKS-2982] Fix missing echo post feedback

### PR DESCRIPTION
**Pull request by mhuynh5757**
## Submission Checklist

 - [x] Updated the `CHANGELOG.md` to reflect any **feature**, **bug fixes**, or **known issues** made in the source code
 - [x] Tested code changes and observed expected behavior in the example app
 - [x] Performed a visual inspection of the `Files changed` tab prior to submitting the pull request for review to ensure proper usage of the style guide

> All third-party contributors acknowledge that any contributions they provide will be made under the same open-source license that the open-source project is provided under.

- [x] I acknowledge that all my contributions will be made under the project's license.

## Description

This PR uses common constants to facilitate the native layer interface for the post feedback method.

## Breakdown

- Use common constants for post feedback values.
- Add mappings to each layer for these common constants to the appropriate enum value.

## Validation

- Unit tests.
- Manual testing on iOS and Android.

## Additional Notes

N/A

### CODE REVIEW

File: android/src/main/java/com/twiliovoicereactnative/TwilioVoiceReactNativeModule.java [hunk 1]
```diff
@@ -540,39 +541,53 @@ public void callInvite_reject(String uuid, Promise promise) {
...
+    return scoreMap.containsKey(score)
+      ? scoreMap.get(score)
+      : Call.Score.NOT_REPORTED;
   }
 
+  /**
+   * Map of common constant issue strings to the Call.Issue enum.
+   */
+  private final Map<String, Call.Issue> issueMap = Map.of(
+    CommonConstants.CallFeedbackIssueAudioLatency, Call.Issue.AUDIO_LATENCY,
```
Comment (afalls-twilio) [inline android/src/main/java/com/twiliovoicereactnative/TwilioVoiceReactNativeModule.java:572]: Move this to where the other private variables are defined and make it static as well (it never changes correct?)

File: ios/TwilioVoiceReactNative.m [hunk 2]
```diff
@@ -996,22 +996,40 @@ - (NSString *)stringOfState:(TVOCallState)state {
...
+    return TVOCallFeedbackScoreTwoPoints;
+  } else if ([score isEqualToString:kTwilioVoiceReactNativeCallFeedbackScoreThree]) {
+    return TVOCallFeedbackScoreThreePoints;
+  } else if ([score isEqualToString:kTwilioVoiceReactNativeCallFeedbackScoreFour]) {
+    return TVOCallFeedbackScoreFourPoints;
+  } else if ([score isEqualToString:kTwilioVoiceReactNativeCallFeedbackScoreFive]) {
+    return TVOCallFeedbackScoreFivePoints;
+  }
+  return TVOCallFeedbackScoreNotReported;
+}
```
Comment (bobiechen-twilio) [inline ios/TwilioVoiceReactNative.m:1014]: 4-space indentation

**Review by bobiechen-twilio — APPROVED** [review]
iOS changes look good overall, just a minor comment regarding indentation

File: api/voice-react-native-sdk.api.md [hunk 3]
```diff
@@ -255,19 +251,19 @@ export namespace Call {
...
-        NotReported = 0,
-        One = 1,
-        Three = 3,
-        Two = 2
+        'Five' = "five",
+        'Four' = "four",
+        'NotReported' = "not-reported",
+        'One' = "one",
+        'Three' = "three",
+        'Two' = "two"
```
Comment (charliesantos) [inline api/voice-react-native-sdk.api.md:266]: This enum is public right?

Comment (mhuynh5757) [reply api/voice-react-native-sdk.api.md:266]: Yes, that's correct. This is a public enum. Are you concerned about this being a breaking change?

**Review by charliesantos — APPROVED** [review]

**Review by afalls-twilio — APPROVED** [review]
LGTM

**Review by kpchoy — APPROVED** [review]

**Pull request merged**